package.json

```diff
@@ -27,5 +27,6 @@
         "url": "https://github.com/Microsoft/tslib.git"
     },
     "main": "tslib.js",
+    "jsnext:main": "tslib.es6.js",
```

Comment:
Maybe use `module` instead of `jsnext:main` to achieve same behavior, but less configuration since the `module` has been set to `true` as default in **rollup-plugin-node-resolve**.